### ISSUE TITLE
dep-scan does not detect python `requirements.txt` vulnerabilities 

### ISSUE THREAD

**Issue by torque59** [AppThreat/dep-scan#23]
Neat work with the dep-scan :). 
I just wanted to file an issue, which seems like a potential bug.

**Issue:**

While trying out against a simple `requirements.txt` file with the content, it detects as no vulnerabilities, while not being the same.
```
Django==2.2.8
```

**Sample Output:**

Command Ran: 
```
export SCAN_DEBUG_MODE="debug"
scan -i ./
```` 

```
DEBUG [2020-09-29 22:23:46,548] Scanning using the bom file ./reports/bom-python.json
INFO [2020-09-29 22:23:46,672] No license violation detected ✅
DEBUG [2020-09-29 22:23:49,974] Vulnerability database loaded from /Users/xxxx/Library/Application Support/vdb/data.vdb
INFO [2020-09-29 22:23:49,975] To use GitHub advisory source please set the environment variable GITHUB_TOKEN!
DEBUG [2020-09-29 22:23:53,049] Vulnerability database contains 1948391 records
INFO [2020-09-29 22:23:53,049] Performing regular scan for ./ using plugin python
INFO [2020-09-29 22:23:53,049] Scanning 1 oss dependencies for issues
INFO [2020-09-29 22:24:39,343] No oss vulnerabilities detected ✅
```

This returns no vulnerabilities, well which is not the case as i did verify with `vdb --search Django:2.2.8 ` and it has vulnerabilities.

Let me know if i'm missing something here or got the options wrong.

P.S: I tried debugging the issue, but it seems to point to the dependency vulnerability-db -> `bulk_index_search` returning NULL from what i could check.

**Comment by prabhu** [comment]
Hi @torque59 

Thank you :) This is an interesting find!

Running depscan cli directly is working fine for me

```
SCAN_DEBUG_MODE=debug python3 depscan/cli.py -i ~/sandbox/depcasetest
```

But I'm able to reproduce the error when this is invoked via the scan container image. The bom json files look identical so something else must be going wrong. Will investigate further.

**Comment by prabhu** [comment]
Fixed in v1.8.6. Required an additional alias since the vendor for django is set as `djangoproject`. In general, the quality of NVD database is quite average. In a file called [normalize.py](https://github.com/AppThreat/dep-scan/blob/master/depscan/lib/normalize.py#L28) several variations of the given package vendor combination would be created to identify and match the package names. Setting github_token would have helped pull some amount of data from github advisories which correctly uses the vendor name `pip` for django.

**Comment by torque59** [comment]
@prabhu Nice i was wondering the same, when i actually debugged with the variations being generated. Thanks for the quick fixes :)
